Source/DafnyLanguageServer/Workspace/IDocumentDatabase.cs

```diff
@@ -57,6 +57,13 @@ public interface IDocumentDatabase {
     /// <returns>An instance of the managed document, <c>null</c> if the specified document was not found.</returns>
     Task<DafnyDocument?> GetDocumentAsync(TextDocumentIdentifier documentId);
 
+    /// <summary>
+    /// Gets the last version of this document
+    /// </summary>
+    /// <param name="documentId">The ID of the document to resolve.</param>
+    /// <returns>An instance of the managed document, <c>null</c> if the specified document was not found.</returns>
+    Task<DafnyDocument?> GetLatestDocumentAsync(TextDocumentIdentifier documentId);
```

Comment:
I don't understand why you'd need this from the outside.

Reply:
> Why do you need this?

Because when I was developing this PR, there wasn't a `GetLastDocumentAsync` so I had to create one myself.
Now with the latest master there is the equivalent of the method I needed, so this conversation is obsolete.